go/libraries/doltcore/diff/querydiff/query_differ_test.go

```diff
@@ -262,3 +362,506 @@ func testQueryDiffer(t *testing.T, test queryDifferTest) {
 	assert.Nil(t, to)
 	assert.Equal(t, io.EOF, err)
 }
+
+// subset of test data from go-mysql-server/enginetest/testdata.go:CreateSubsetTestData()
+var engineTestSetup = []testCommand{
```

Comment:
Probably should just create these with a doltHarness. Not sure what value you get by duplicating them here.

Reply:
That method even lets you create a subset of the tables you are interested in (but creating all of them should work)